### PR TITLE
TLC-529 axios userGetByEmail added to register.js

### PR DESCRIPTION
**Pull request by aliakseyevavikt**

### CODE REVIEW

File: 20605/specs/smoke/register.js [hunk 1]
```diff
@@ -10,4 +11,9 @@ describe('REGISTER', function() {
     RegisterStep2Page.registerUserStep2(newUser);
     expect((ProfilePage.headerProfile).getText()).eq(newUser.firstName + ' ' + newUser.lastName);
   });
-});
\ No newline at end of file
+
+  it('API check the new user', async () => {
```
Comment (Alina-Marukhnenko) [inline 20605/specs/smoke/register.js:15]: add TC number)

File: 20605/testData/register.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const newUser = {
   lastName: faker.name.lastName(),
   email: faker.internet.email(),
   password: faker.internet.password(),
-  phone: faker.phone.phoneNumber('##########') //10
+  phone: faker.phone.phoneNumber('###########') //11
```
Comment (Alina-Marukhnenko) [inline 20605/testData/register.js:8]: comment //11?

File: 20605/specs/smoke/register.js (hunk 1 above)
Comment (aliakseyevavikt) [reply 20605/specs/smoke/register.js:15]: The TC is the same as above. It's the additional test, not sure if I need to add TC number.

File: 20605/testData/register.js (hunk 2 above)
Comment (aliakseyevavikt) [reply 20605/testData/register.js:8]: I didn't find how to change the format (phone number). I add the comment in order to not count "#" every time.

**Pull request merged**